go-controller/pkg/network-controller-manager/network_controller_manager.go

```diff
@@ -184,6 +184,7 @@ func NewNetworkControllerManager(ovnClient *util.OVNClientset, wf *factory.Watch
 			CloudNetworkClient:   ovnClient.CloudNetworkClient,
 			EgressServiceClient:  ovnClient.EgressServiceClient,
 			APBRouteClient:       ovnClient.AdminPolicyRouteClient,
+			EgressQoSClient:      ovnClient.EgressQoSClient,
```

Comment:
hmm is Master a.k.a ovnkube-controller starting to patch stuff for QoS? isn't it only ClusterManager?

Reply:
yes, it patches for `status.Messages`.